.github/workflows/tests.yml

```diff
@@ -81,7 +81,8 @@ jobs:
       TSAN_OPTIONS: "verbosity=1:halt_on_error=1:suppressions=/code/faabric/thread-sanitizer-ignorelist.txt:history_size=7"
       UBSAN_OPTIONS: "print_stacktrace=1:halt_on_error=1"
     container:
-      image: faasm/faabric:0.3.1
+      image: faasm/faabric:0.3.2
+      options: --privileged
```

Comment:
Needed to actually run the `userfaultfd` tracking.

Reply:
Another option is setting the `/proc/sys/vm/unprivileged_userfaultfd` sysctl to 1

Reply:
But probably not an option on github actions runners actually

Reply:
Ah nice, yes this is a bit of a blunt instrument. We also use `privileged` in Faasm for cgroups, so it's a bit lazy but works across the board.